src/PIL/EpsImagePlugin.py

```diff
@@ -117,12 +117,15 @@ def Ghostscript(tile, size, fp, scale=1, transparency=False):
 
     device = "pngalpha" if transparency else "ppmraw"
 
+    w, h = size
```

Comment:
Minor concern: declaring `w` and `h` here seems a bit odd to me, since the width and height have already been used earlier in
https://github.com/python-pillow/Pillow/blob/8a13bf8b936497303f028b458483f2e8a64e96c6/src/PIL/EpsImagePlugin.py#L85-L86

Reply:
Yeah, I was thinking about that but didn't want to make a more intrusive change here. If you like, I can, though (though it'll conflict with https://github.com/python-pillow/Pillow/pull/7527/files#diff-c43e9ae25942c60787ebabdbfabddf8618c31d166f93520ea22c8547957f7e9eL80 so I'd love for that to get merged first :) ).

Reply:
#7527 has now been merged